### PR TITLE
Add wiki info to contrib docs

### PR DESCRIPTION
**Pull request by crandmck**
This addresses https://github.com/strongloop/loopback/issues/2776
It also reorganizes the "Contributing" sidebar.

@superkhau

### CODE REVIEW

**Review by superkhau — APPROVED** [review]

**Comment by superkhau** [comment]
Please send follow up PRs when you do the actual deprecation + gg announcements. Follow up PRs should be to loopback-example and loopback-contrib-docs to point at the new doc site.

**Comment by crandmck** [comment]
https://github.com/strongloop/loopback/issues/2776 says _you_ will update the READMEs and do deprecations.  Please do that.  I don't have time.

File: pages/en/contrib/code-contrib.md [hunk 1]
```diff
@@ -1,17 +1,109 @@
...
+
+All LoopBack projects follow the same convention for running tests. To run a projects test suite, change to the project's directory and run:
+
+```sh
+npm test
+```
+
+If a test is failing, you should open an issue on the appropriate repositories GitHub issues page.
+
+#### Signing patches
```
Comment (bajtos) [inline pages/en/contrib/code-contrib.md:78]: Signing the patches is no longer needed. We are collecting CLAs through the clahub app. This whole section can be removed (or replaced with text describing how to sign the CLA through the web app).

Comment (crandmck) [reply pages/en/contrib/code-contrib.md:78]: @bajtos Thanks for catching that.  Removed by 478c7dec04e2251035b0cfd72e8490bd5f24a095.

**Pull request merged**